src/mlpack/core/optimizers/saga/saga_update.hpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file saga_update.hpp
+ * @author Prabhat Sharma
+ *
+ * Vanilla update for stochastic variance reduced gradient (SVRG).
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_SAGA_SAGA_UPDATE_HPP
+#define MLPACK_CORE_OPTIMIZERS_SAGA_SAGA_UPDATE_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+  namespace optimization {
+
+/**
+ * Vanilla update policy for Stochastic variance reduced gradient (SVRG).
```

Comment:
Vanilla update for SAGA. I think the comments needs to be updated too. Check them out..